src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -47,7 +50,8 @@ internal class NuGetPackageDownloader : INuGetPackageDownloader
             IReporter reporter = null,
             RestoreActionConfig restoreActionConfig = null,
             Func<IEnumerable<Task>> timer = null,
-            bool verifySignatures = false)
+            bool verifySignatures = false,
```

Comment:
I'm guessing we never actually require the tool packages to be signed, but how do users trust them? Probably something to discuss for another PR - I'd be curious to know whether there was a threat model for dotnet tools.

Reply:
Nuget has commands for managing the nuget trust store - see 'dotnet nuget trust ---help' for examples of that.

Reply:
There are multiple parts to signature verification. 
1. Does a package have a signature, is it valid?
2. Do I trust this package's signer. 

The `dotnet nuget trust `  command handles the latter part. It basically allows you to "limit" the number of signers you trust by virtue of only allowing certain author/repositories/certificates.

For #1, NuGet *always* verifies the signature regardless of what's enabled through nuget trust. The verification happens on windows, and @dtivel recently did the work to enable that verification on linux.

Reply:
Thanks for the comment! To be follow up in #35283